frontend/src/Components/LoginSignup/SignupVolunteerSubForm.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useEffect} from 'react';
+import axios from 'axios';
+
+export default function SignupAdminSubForm(props) {
+    // const [ skillsList, setSkillsList ] = useState([]);
+    const [ skillsList, setSkillsList ] = useState([{'skill_id':1, 'skill':'skill1'}, {'skill_id':2, 'skill':'skill2'}, {'skill_id':3, 'skill':'skill3'}]);
```

Comment:
any reason you use dummy data in an array as the initial state here instead of an empty array? visual rendering?

Reply:
resolved in next pull req

Reply:
yes, because it that stage (when the pr was made), skills routes was not up yet
this has been addressed in the next PR